crates/biome_js_analyze/src/lint/nursery/use_sorted_classes/tailwind_preset.rs

```diff
@@ -572,11 +573,684 @@ const UTILITIES_LAYER_CLASSES: [&str; 569] = [
     "duration-",
     "ease-",
     "will-change-",
+    "contain-none$",
```

Comment:
Where are these coming from? AFAIK, this is a plugin and the codegen script works with a clean, empty config. I couldn't find the source.

Reply:
Actually, I am just doing `just gen-tw` in order to execute the script and generate the `preset`, I had to install bun and `bun install` the package, so maybe those were included in some newer version of TailwindCSS?